deprecated/packages/transifex-client/transifex-client.nuspec

```diff
@@ -18,18 +18,11 @@
     <tags>transifex-client transifex translation cli cross-platform foss</tags>
```

Comment:
I didn't mention it (mainly because I forgot, and we don't have any documentation of it).

But would it make sense if we replace all of the tags to just be `deprecated`?

Thoughs @pauby @vexx32

Reply:
No idea tbh. I guess it kind of makes sense that we'd want to reduce the discoverability of the package? Not sure.

Reply:
After thinking for a bit I think this makes sense, I've updated this too.